packages/nextjs-contentful-renderer/config/jest/setupTests.ts

```diff
@@ -0,0 +1,4 @@
+process.env.CONTENTFUL_SPACE_ID = 'space';
+process.env.CONTENTFUL_ACCESS_TOKEN_PREVIEW = 'preview_token';
+process.env.CONTENTFUL_ACCESS_TOKEN_DELIVERY = 'delivery_token';
+process.env.SITE_DOMAIN = 'site_domain';
```

Comment:
Is this needed because jest doesn't have access to read `.env` file?

Reply:
yup